sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -278,19 +279,61 @@ def test_predict_pred_interval(Forecaster, fh, alpha):
         f = _construct_instance(Forecaster)
         y_train = _make_series(n_columns=n_columns)
         f.fit(y_train, fh=fh)
-        if f.get_tag("capability:pred_int"):
-            y_pred, pred_ints = f.predict(return_pred_int=True, alpha=alpha)
-            _check_pred_ints(pred_ints, y_train, y_pred, fh)
-
+        y_pred = f.predict(alpha=alpha)
+        if not f._has_predict_quantiles_been_refactored():
+            with pytest.raises(NotImplementedError):
+                pred_ints = f.predict_interval(fh)
+                _check_pred_ints(pred_ints, y_train, y_pred, fh)
         else:
-            with pytest.raises(NotImplementedError, match="prediction intervals"):
-                f.predict(return_pred_int=True, alpha=alpha)
+            f.predict(alpha=alpha)
```

Comment:
I'm confused about this line - should this not be `f.predict_interval`?

Reply:
we also need to do something to check the return. It should be in the expected format.